App/src/main/java/com/gluonhq/chat/views/ChannelCell.java

```diff
@@ -12,13 +12,18 @@ public ChannelCell() {
         tile = new ListTile();
         setGraphic(tile);
         setText(null);
+        getStyleClass().add("focus-cell");
```

Comment:
It is better as a pseudo class, like:
```
private static final PseudoClass PSEUDO_CLASS_UNREAD = PseudoClass.getPseudoClass("unread");


@Override
    public void updateItem(Channel channel, boolean empty) {
        super.updateItem(channel, empty);

        pseudoClassStateChanged(PSEUDO_CLASS_UNREAD, channel!= null && channel.isUnread());
    ...
```
with css:

.channel-list .channel-cell:unread .primary-text {
    -fx-font-weight: bold;
}